### DISCUSSION TITLE
Doubt baits and target capture

### DISCUSSION THREAD

**Discussion by ghost** [etal/cnvkit#612] [Q&A]
Hi,

I am confused with the step **target**. The positional argument of the **cnvkit.py target** tells us "BED or interval file listing the targeted regions.", that I guess it is my target capture (not the probes/baits), but in the documentation (https://cnvkit.readthedocs.io/en/stable/pipeline.html#target) it says the next "The BED file should be the baited genomic regions for your target capture kit, as provided by your vendor.", so I am a bit confused which one to use. If it would be the baits, there is no step where I introduce my target bed file, this sound quite confusing to me.

Best,

Roberto

**Comment by tetedange13** [comment]
Hi @ralonso-igenomix,
<br>

Not an author of CNVkit, but here is my understanding:
- "Targets" here is meant as "therorical regions", with genomic coordinates matching what you can find in databases (UCSC table brower, refSeq...)
- And "baits/probes" are "captured regions in reality", depending on your kit and vendor design
=> Most of the time, "baits" coordinates match exonic/intronic coordinates, but with a bit of extra bases left and right (you also have cases with a probe matching several exons, but anyway)
=> What matters is that this "baits" BED should theorically be **closer to what you really sequenced and what you have in your BAM files**

So IMHO the most important sentence from "target" part of documentation is the following:
> But CNVkit will work much better if you have the real capture region coordinates – contact your sequencing lab for help with this step.

=> Meaning that the **BED you should give as argument for `cnvkit.py target` is the "baits/probes" one**, in order for CNVkit to give you the best results possible
=> This subcommand is called `target`, but probably by opposition to the `antitarget` subcommand
<br>

Hope this helps
Have a nice day.
Felix

**Reply by ghost** [reply]
Hi Felix,

thanks for your well explained answer. I understood that the baits file is only the probes, but my real capture (not taking into account theorical regions such us UCSC tablebrowser) is the extended baits due to sequencing, so I though there should be two files when you do an exome capture, the baits file and the real targets file, I may be wrong. 
In any case, I did the analysis with my targets file (because is the only one that I have), and the antitarget part got most of the computational time and strange results, such us cnvs with n=40000, but if I don't split, I get much more meaningful results, cnv with n of 40.

Many thanks,

Roberto
